Kitodo/src/test/java/de/sub/goobi/helper/encryption/MD4Test.java

```diff
@@ -38,16 +37,12 @@
     }
 
     @Test
-    public void encryptTest() {
+    public void encryptTest() throws Exception {
```

Comment:
I think for tests using more specific exception is not necessary. In this way method is shorter.